src/components/PersonInfo/PersonInfo.tsx

```diff
@@ -0,0 +1,50 @@
+import { Link, useParams } from 'react-router-dom';
+import { Person } from '../../types';
+import classNames from 'classnames';
+
+type Props = {
+  person: Person;
+  father?: Person;
+  mother?: Person;
+};
+
+export const PersonInfo: React.FC<Props> = ({ person, father, mother }) => {
+  const { slug } = useParams();
+
+  return (
+    <tr
+      key={person.slug}
```

Comment:
The `key` prop is unnecessary here since this `tr` element is not part of a list directly rendered by a `map` function. The `key` prop should be used in the parent component where the list is being mapped over, such as in `PeopleTable.tsx`.